frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -152,19 +154,27 @@ const MutableRow: React.FC<MutableRowProps> = ({ data, onUpdate, onReturn, valid
   const updateCallback = (e: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) =>
     error ? () => {} : onUpdate(e);
 
+  const disabledTextFieldComponent = (
+    <TextField
+      disabled
+      id={data.id}
+      name={data.name}
+      defaultValue={data.value}
+      label={data.textFieldLabels?.disabledField ?? data.textFieldLabels?.disabledField}
```

Comment:
Should the second value be something else?
```suggestion
      label={data.textFieldLabels?.disabledField}
```

Reply:
shawna & synced in standup as i originally wrote that. yeah we can change to the suggestion or `data.textFieldLabels?.disabledField ?? ""`